pages/fpki_cas.md

```diff
@@ -73,7 +73,9 @@ We realize all the acronyms and labels may be confusing and welcome your input t
 |-----------|---------------|
 | PKI Shared Service Provider (SSP) Certification Authorities | An SSP CA is *subordinate* to the FCPCA.  Any certificate that an SSP CA creates, signs, and issues to people or devices is in the FCPCA _trust chain_. An SSP must adhere to strict federal IT security standards and requirements.  The SSPs are granted a FISMA Authority To Operate (ATO), undergo continuous monitoring, and are contracted by the Federal Government to issue certificates to federal employees and contractors, as well as devices that are deployed in federal agency networks. |
 | Private Sector Certification Authorities | A Private Sector CA that is cross-certified has shown a valid need to conduct business or provide PKI services to the Federal Government. |
-| Access Certificates for Electronic Services (ACES) Certification Authorities | The ACES CAs issue certificates to authorized U.S. business representatives who need to digitally sign documents or access certain web-based systems.  ACES was established in the late 1990s and is the predecessor for the development of today's Federal PKI. |
+| Access Certificates for Electronic Services (ACES) Certification Authorities | The ACES CAs issue certificates to authorized U.S. business representatives who need to digitally sign documents or access certain web-based systems.  ACES was established in the late 1990s and is the predecessor for the development of today's Federal PKI. See note below on ACES CA and credential deprecation. |
 | Other Government Certification Authorities | These CAs are managed and operated by State, Local, Tribal, Territorial, or International Governments. |
 | Bridge Certification Authorities | Bridge CAs connect member PKIs and are designed to enable interoperability between different PKIs operating under their own certificate policies. A Bridge CA is not a _root_. |
 | Federal Agency Legacy | Prior to 2004, some agencies had already deployed and invested in their own PKI and CAs. Some of these agencies opted out of migrating to the SSP Program and continued to manage their existing infrastructures. These Federal Agencies Legacy operate one or more CAs that are cross-certified with a Federal PKI Trust Infrastructure CA.|
+
+{% include alert-warning.html content="ACES credentials are deprecated for Private Sector Certification Authorities or Bridge Certification Authorities. For more information, see the [GSA ACES Sunset FAQ Sheet] (https://www.idmanagement.gov/wp-content/uploads/sites/1171/uploads/gsa-aces-sunset-guide.pdf) or email the [GSA ACES Program] (GSA-ACES@GSA.gov.)" %}
```

Comment:
@weirdscience look at the preview for this item?  Since this is an alert and in html - it won't process markdown syntax.  The link and the email address don't resolve.

Reply:
No worries, I'll fix it. Learn something new every day.